filament/include/filament/IndirectLight.h

```diff
@@ -342,6 +348,80 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
     /** @deprecated use static versions instead */
     UTILS_DEPRECATED
     math::float4 getColorEstimate(math::float3 direction) const noexcept;
+
+    /**
+     * Gets IBL type.
+     */
+    IndirectLight::Mode getTechnique() const noexcept;
+
+    /**
+     * Sets IBL type to sphere.
+     *
+     * @param The center (.xyz) and radius (.w) of the IBL sphere proxy geometry
+     */
+    void setSphereProxy(const filament::math::float4& sphere);
+
+    /**
+     * Gets the properties of sphere IBL.
+     *
+     * @return The center (.xyz) and radius (.w) of the IBL sphere proxy geometry
+     */
+    filament::math::float4 getSphereProxy() const;
+
+    /**
+     * Sets IBL type to box.
+     *
+     * @param The IBL box proxy geometry
+     */
+    void setBoxProxy(const filament::Box& box);
+
+    /**
+     * Gets the properties of box IBL.
+     *
+     * @return The filament::Box representing the the IBL box proxy geometry
+     */
+    filament::Box getBoxProxy() const;
+
+    /**
+     * Reverts back to infinite IBLs.
+     */
+    void clearProxy();
+
+    /**
+     * Sets the center of the finite IBL proxy geometry.
+     *
+     * @param iblCenter The center of the finite IBL proxy geometry
+     */
+    void setCenter(const math::float3& inIblCenter) noexcept;
+
+    /**
+     * Gets IBL center.
+     */
+    const math::float3& getCenter() const noexcept;
+
+    /**
+     * Sets the half extents of the finite IBL proxy geometry.
+     *
+     * @param iblCenter The half extents of the finite IBL proxy geometry
+     */
+    void setHalfExtents(const math::float3& inIblHalfExtents) noexcept;
+
+    /**
+     * Gets IBL half extents.
+     */
+    const math::float3& getHalfExtents() const noexcept;
+
```

Comment:
Remove these API, they're not needed anymore.
(you can keep the getters on the `FIndirectLight` side)